korman/properties/modifiers/anim.py

```diff
@@ -143,6 +143,15 @@ def post_export(self, exporter, bo, so):
             # If the mass is zero, then we will fail to animate. Fix that.
             if phys.mass == 0.0:
                 phys.mass = 1.0
+                
+                # On CC and Prime, set kPinned so it doesn't fall through
+                if bpy.context.scene.world.plasma_age.version in ["pvPots", "pvPrime"]:
+                    sim.setProperty(plSimulationInterface.kPinned, True)
+                    phys.setProperty(plSimulationInterface.kPinned, True)
+        
+        # Do the same for children objects
+        for child in so.coord.object.children:
+            self.post_export(exporter, None, child.object)
```

Comment:
This would work, but I am a little apprehensive about this design. You've assumed here that it is safe to call `post_export` on a modifier more than once. In the future, this `post_export` method might be modified in ways that make it unsafe to call recursively. Or, it might depend on an object having an AGModifier or ATCAnim, which the child objects may not have. I would prefer to see the physics settings stripped into another method and applied from here.